src/Data/Time/Format/Human.hs

```diff
@@ -16,7 +16,6 @@ module Data.Time.Format.Human
     ) where
 
 import Data.Time
-
```

Comment:
There still this and 1 other unexpected whitespace change here, can you remove them?